test/controllers/events_controller_test.rb

```diff
@@ -112,7 +112,21 @@ def setup_live_events
       assert_match("data-name='30d' selected", @response.body)
     end
 
-    test "selects correct aggregation from dropdown for day" do
+    test "selects correct date range when none is selected" do
```

Comment:
Can you also add a test where there's a random value for `?date=`, like `"?date=blabla"` to test that.